### ISSUE TITLE
Add a how to contribute file

### ISSUE THREAD

**Issue by codecakes** [Xcov19/mycovidconnect#77]
See [How to create a contribution file](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file)

Move the instructions to this file from the README file;

**Comment by Haniket** [comment]
I want to work on it................

**Comment by Haniket** [comment]
which part of readme file should be moved???..........

**Comment by dheeraj-tripathi** [comment]
I would like to contribute as well, would you mind assigning it to me?

**Comment by codecakes** [comment]
@Haniket [this part](https://github.com/Xcov19/mycovidconnect#start-here-project-setup-guidelines) should be part of the contribution; when you set the project locally, see https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors#adding-a-contributing-file

Mention what relevant skillsets are required like: nodejs, reactjs, basic knowledge of docker;

Under the contribution, guide the devs that the following file should be added by the dev for now as `.git/hooks/pre-commit`
 we will update pre commit mechanism later:
```bash
#!/bin/sh
#
# An example hook script to verify what is about to be committed.
# Called by "git commit" with no arguments.  The hook should
# exit with non-zero status after issuing an appropriate message if
# it wants to stop the commit.
#
# To enable this hook, rename this file to "pre-commit".
if git rev-parse --verify HEAD >/dev/null 2>&1
then
	against=HEAD
else
	# Initial commit: diff against an empty tree object
	against=4b825dc642cb6eb9a060e54bf8d69288fbee4904
fi
# If you want to allow non-ASCII filenames set this variable to true.
allownonascii=$(git config --bool hooks.allownonascii)
# Redirect output to stderr.
exec 1>&2
# Cross platform projects tend to avoid non-ASCII filenames; prevent
# them from being added to the repository. We exploit the fact that the
# printable range starts at the space character and ends with tilde.
if [ "$allownonascii" != "true" ] &&
	# Note that the use of brackets around a tr range is ok here, (it's
	# even required, for portability to Solaris 10's /usr/bin/tr), since
	# the square bracket bytes happen to fall in the designated range.
	test $(git diff --cached --name-only --diff-filter=A -z $against |
	  LC_ALL=C tr -d '[ -~]\0' | wc -c) != 0
then
	cat <<\EOF
Error: Attempt to add a non-ASCII file name.
This can cause problems if you want to work with people on other platforms.
To be portable it is advisable to rename the file.
If you know what you are doing you can disable this check using:
  git config hooks.allownonascii true
EOF
	exit 1
fi
# If there are whitespace errors, print the offending file names and fail.
exec git diff-index --check --cached $against --
STAGED_FILES=$(git diff --cached --name-only --diff-filter=ACM | grep ".jsx\{0,1\}$")
ESLINT="$(git rev-parse --show-toplevel)/node_modules/.bin/eslint"
if [[ "$STAGED_FILES" = "" ]]; then
  exit 0
fi
PASS=true
printf "\nValidating Javascript:\n"
# Check for eslint
if [[ ! -x "$ESLINT" ]]; then
  printf "\t\033[41mPlease install ESlint\033[0m (npm i --save-dev eslint)"
  exit 1
fi
for FILE in $STAGED_FILES
do
  "$ESLINT" "$FILE"
  if [[ "$?" == 0 ]]; then
    printf "\t\033[32mESLint Passed: $FILE\033[0m"
  else
    printf "\t\033[41mESLint Failed: $FILE\033[0m"
    PASS=false
  fi
done
printf "\nJavascript validation completed!\n"
if ! $PASS; then
  printf "\033[41mCOMMIT FAILED:\033[0m Your commit contains files that should pass ESLint but do not. Please fix the ESLint errors and try again.\n"
  exit 1
else
  printf "\033[42mCOMMIT SUCCEEDED\033[0m\n"
fi
exit $?
``` 

Feel free to send a PR for this linking this issue or send any questions.

**Comment by manasi999** [comment]
hello @codecakes could i contribute to this as well ?

**Comment by codecakes** [comment]
pls send a pr and add a comment refering to #77, something like "fixes #77"

Look at CONTRIBUTING file; it should also mention:
- make sure you have first merged changes from master;
- then send a PR of ur branch
